File: src/main/java/pl/yalgrin/playnite/simplesync/repository/PlatformDiffRepository.java
package pl.yalgrin.playnite.simplesync.repository;

import org.springframework.data.r2dbc.repository.R2dbcRepository;
import org.springframework.stereotype.Repository;
import pl.yalgrin.playnite.simplesync.domain.PlatformDiff;

@Repository
public interface PlatformDiffRepository extends R2dbcRepository<PlatformDiff, Long> {
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/repository/PlatformDiffRepository.kt
package pl.yalgrin.playnite.simplesync.library.repository

import org.springframework.data.r2dbc.repository.R2dbcRepository
import org.springframework.stereotype.Repository
import pl.yalgrin.playnite.simplesync.library.domain.PlatformDiff

@Repository
interface PlatformDiffRepository : R2dbcRepository<PlatformDiff, Long>